src/itree.cpp

```diff
@@ -122,6 +122,11 @@ int main(int argc, char** argv){
         auto path = correct_path(vm, env);
         irods::experimental::client_connection conn;
 
+        if (!fs::client::is_collection(conn, path)) {
+            std::cerr << fmt::format("Error: logical path [{}] does not refer to a collection.\n", path.c_str());
```

Comment:
I don't think we need to report the actual path here. Alternatively, you can remove the `logical path ` from the message since the word, `collection`, implies a logical path.

Reply:
Okey doke, that is done.